tests/integration_tests/explore/form_data/api_tests.py

```diff
@@ -67,65 +67,91 @@ def dataset_id() -> int:
         return dataset.id
 
 
+@pytest.fixture
```

Comment:
Maybe have one fixture that returns the `datasource` with both `id` and `type`?

Reply:
yeah, good call